projects/packages/blaze/src/class-dashboard-config-data.php

```diff
@@ -0,0 +1,147 @@
+<?php
+/**
+ * Blaze dashboard Initial State
+ *
+ * @package automattic/jetpack-blaze
+ */
+
+namespace Automattic\Jetpack\Blaze;
+
+use Automattic\Jetpack\Current_Plan;
+use Automattic\Jetpack\Modules;
+use Jetpack_Options;
+
+/**
+ * Class Dashboard_Config_Data
+ */
+class Dashboard_Config_Data {
+	/**
+	 * Set configData to window.configData.
+	 *
+	 * @param array $config_data The config data.
+	 */
+	public function get_js_config_data( $config_data = null ) {
+		return 'window.configData = ' . wp_json_encode(
+			$config_data === null ? $this->get_data() : $config_data
+		) . ';';
+	}
+
+	/**
+	 * Return the config for the app.
+	 */
+	public function get_data() {
+		$blog_id      = Jetpack_Options::get_option( 'id' );
+		$empty_object = json_decode( '{}' );
+		return array(
+			'admin_page_base'          => $this->get_admin_path(),
+			'api_root'                 => esc_url_raw( rest_url() ),
+			'blog_id'                  => Jetpack_Options::get_option( 'id' ),
+			'enable_all_sections'      => false,
+			'env_id'                   => 'production',
+			'google_analytics_key'     => 'UA-10673494-15',
+			'hostname'                 => wp_parse_url( get_site_url(), PHP_URL_HOST ),
+			'i18n_default_locale_slug' => 'en',
+			'mc_analytics_enabled'     => false,
+			'meta'                     => array(),
+			'nonce'                    => wp_create_nonce( 'wp_rest' ),
+			'site_name'                => \get_bloginfo( 'name' ),
+			'sections'                 => array(),
+			// Features are inlined in Calypso Blaze app (wp-calypso/apps/blaze-dashboard)
+			'features'                 => array(),
+			'intial_state'             => array(
+				'currentUser' => array(
+					'id'           => 1000,
+					'user'         => array(
+						'ID'         => 1000,
+						'username'   => 'no-user',
+						'localeSlug' => $this->get_site_locale(),
+					),
+					'capabilities' => array(
+						"$blog_id" => $this->get_current_user_capabilities(),
+					),
+				),
+				'sites'       => array(
+					'items'    => array(
+						"$blog_id" => array(
+							'ID'           => $blog_id,
+							'URL'          => site_url(),
+							'jetpack'      => true,
+							'visible'      => true,
+							'capabilities' => $empty_object,
+							'products'     => array(),
+							'plan'         => $empty_object, // we need this empty object, otherwise the front end would crash on insight page.
+							'options'      => array(
+								'wordads'    => ( new Modules() )->is_active( 'wordads' ),
```

Comment:
I'm not sure I understand why we need the connection to the WordAds module here and downer. Shouldn't Blaze work even when the module is disabled?

Reply:
Let me check this point; I don't think we need wordads enabled for Blaze. Some of the end points in the DSP have the wordads prefix, but those are under the DSP, they probably don't have anything to do with that module.

Reply:
Fixed in a20893ce4aba6dcee51413949e06a43c86d6d14e